clippy_lints/src/trivially_copy_pass_by_ref.rs

```diff
@@ -118,7 +118,7 @@ impl<'a, 'tcx> TriviallyCopyPassByRef {
                         cx,
                         TRIVIALLY_COPY_PASS_BY_REF,
                         input.span,
-                        "this argument is passed by reference, but would be more efficient if passed by value",
+                        &format!("this argument ({} B) is passed by reference, but would be more efficient if passed by value (limit: {} B)", size, self.limit),
```

Comment:
I think it would be a bit more clear to write out 'byte' instead of the abbreviation.

Reply:
updated